include/ginkgo/core/solver/cb_gmres.hpp

```diff
@@ -144,6 +144,13 @@ class CbGmres : public EnableLinOp<CbGmres<ValueType>>,
      */
     size_type get_krylov_dim() const { return krylov_dim_; }
 
+    /**
+     * Sets the krylov dimension
+     *
+     * @param other  the new krylov dimension
+     */
+    void set_krylov_dim(const size_type &other) { krylov_dim_ = other; }
```

Comment:
I only saw now that we use reference here, I don't think it's really needed for a simple number like that, but that's fine with me.

Reply:
I simply copy-pasted the same function from `GMRES` to `CB-GMRES`.
I agree it is a bit weird to use a reference here, but it should not impact the usage.
I will change it for both to be a regular value. Thanks for mentioning it.